### PR TITLE
Add alternative clean way of creating userStories. Ref #44

### PR DESCRIPTION
**Pull request by Askir**
Removing the old method with 4 variables in favor of a new one with only 2 that has way cleaner code (small methods etc.) and is at least partially unit tested

### CODE REVIEW

File: packages/acceptit-Core.package/ACCompiler.class/instance/compile.in.classified.notifying.ifFail..st [hunk 1]
```diff
@@ -7,6 +7,7 @@ compile: origCode in: cls classified: aCategory notifying: notify ifFail: failBl
 	(match failed)
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Core.package/ACCompiler.class/instance/compile.in.classified.notifying.ifFail..st:7]: I dislike these double negations, consider changing to match succeded

File: packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st [hunk 2]
```diff
@@ -4,6 +4,10 @@ parseTree: aString
 	|evaluator match|
 	evaluator := self grammar synthesizedAttribute: self semantic new.
 	match := self grammar match: aString startingFrom: self treeRule.
-	match succeeded
-		ifTrue: [^evaluator value: match]
-		ifFalse: [^false]       
\ No newline at end of file
+	(match failed == false)
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st:7]: we should be consitend with the way we check match failed. In the first file changed we just used (match failed)

File: packages/acceptit-Core.package/ACUserStory.class/class/definition.st [hunk 3]
```diff
@@ -7,6 +7,6 @@ definition
 			superklass := (super definition findTokens: ' ') first.
 			stream
 				nextPutAll: superklass; space;
-				nextPutAll: 'named:'; space; store: self storyName; crtab;
+				nextPutAll: 'named:'; space; store: self title; crtab;
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Core.package/ACUserStory.class/class/definition.st:10]: This change corrects the class name from the User stories, right?

File: packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st [hunk 4]
```diff
@@ -0,0 +1,8 @@
+constants
+userStoryString
+	
+	^ 'AC Calculator User Story
+	   In order to test my sophisticated calculator class
+	   As a developer
+	   I want to do some complicated calculations'
+	"self userStoryName, String cr, self userStoryContent"
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st:8]: why dont we use the construction in line 8? The hardcoded version could lead to consistancy problems when changing US name and content in their getter methods

File: packages/acceptit-Core.package/ACUserStory.class/class/definition.st (hunk 3 above)
Comment (Askir) [reply packages/acceptit-Core.package/ACUserStory.class/class/definition.st:10]: the naming was just inconsistent. The UserStory title was sometimes named title sometimes storyName. We renamed it to title everywhere.

File: packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st (hunk 4 above)
Comment (Askir) [reply packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st:8]: yes we just forgot about that, thank you.

File: packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st (hunk 2 above)
Comment (Askir) [reply packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st:7]: actually we didn't change anything in the parser except for the added methods. But yes I agree these should be streamlined.

File: packages/acceptit-Core.package/ACCompiler.class/instance/compile.in.classified.notifying.ifFail..st [hunk 5]
```diff
@@ -7,6 +7,7 @@ compile: origCode in: cls classified: aCategory notifying: notify ifFail: failBl
 	(match failed) 
 		ifFalse: [
 			translatedCode := (self buildCode: (evaluator value: match)) readStream.
+			Transcript show: translatedCode contents.
```
Comment (Askir) [inline packages/acceptit-Core.package/ACCompiler.class/instance/compile.in.classified.notifying.ifFail..st:10]: Why is this even here? Should be gone since #40 got fixed.

File: packages/acceptit-Core.package/ACLibrary.class/class/generateNewLibrary..st [hunk 6]
```diff
@@ -10,7 +9,7 @@ generateNewLibrary: aClassOrSymbol
 	
 	newClassName := (aClassOrSymbol asString, 'Library') asSymbol.
 	newCategory := (oldClass == Object)
-		ifTrue: [self libraryCategoryName] 
+		ifTrue: ['Generated']
```
Comment (Askir) [inline packages/acceptit-Core.package/ACLibrary.class/class/generateNewLibrary..st:12]: Why is this here? The branch is rebased to the newest version of develop I don't understand how those change revertions are in this branch....

File: packages/acceptit-Core.package/ACScenarioCompiler.class/instance/buildCode..st [hunk 7]
```diff
@@ -1,4 +1,5 @@
 as yet unclassified
 buildCode: aParsingResult
 
+	Transcript show: aParsingResult.
```
Comment (Askir) [inline packages/acceptit-Core.package/ACScenarioCompiler.class/instance/buildCode..st:4]: Another one from #40

File: packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st (hunk 2 above)
Comment (Askir) [reply packages/acceptit-Core.package/ACScenarioParser.class/instance/parseTree..st:7]: This is also old changed code that got changed back for some reason. I think my commits got broken at some point... I'll have to see through everything and fix it back

File: packages/acceptit-Core.package/ACUserStory.class/class/means.st [hunk 8]
```diff
@@ -1,4 +1,3 @@
-accessing
+constants
 means
-
-	^means
\ No newline at end of file
+	"self subclassResponsibility."
```
Comment (Askir) [inline packages/acceptit-Core.package/ACUserStory.class/class/means.st:3]: If you checkout this branch locally, you can try to uncomment this stuff. But in my experience the self subclass responsibility throws errors while testing because means is implemented later than when the class gets created. Which results in exceptions and failing tests despite everything being created correctly.

File: packages/acceptit-Tests.package/ACGenerateNewLibrary.class/properties.json [hunk 9]
```diff
@@ -7,7 +7,7 @@
 	"commentStamp" : "",
 	"instvars" : [
 		"generatedLibrary" ],
-	"name" : "ACGenerateNewLibraryTest",
+	"name" : "ACGenerateNewLibrary",
```
Comment (Askir) [inline packages/acceptit-Tests.package/ACGenerateNewLibrary.class/properties.json:10]: Another old code change that code reverted somehow
